ambry-server/src/main/java/com/github/ambry/server/AmbryRequests.java

```diff
@@ -508,6 +519,122 @@ public void handleDeleteRequest(NetworkRequest request) throws IOException, Inte
             metrics.deleteBlobTotalTimeInMs, null, null, totalTimeSpent));
   }
 
+  /**
+   * Handles a {@link BatchDeleteRequest} and sends out the response via the {@link RequestResponseChannel}.
```

Comment:
Can you add more info in your comment about what this api does, like handling partial complete and full delete scenario.